src/amo/components/AddonMeta/styles.scss

```diff
@@ -36,3 +36,24 @@
   @include margin-end(-2.5px);
   @include margin-start(-2.5px);
 }
+
+.AddonMeta .MetadataCard-content,
+.AddonMeta .MetadataCard-title {
+  a,
+  a:link,
+  a:visited {
+    color: $grey-50;
+    font-weight: normal;
+    text-decoration: none;
+  }
+
+  a:active,
+  a:hover {
+    color: $link-color;
+  }
+
+  a:focus {
+    outline: 1px dotted $link-color;
+    outline: auto 5px -webkit-focus-ring-color;
```

Comment:
do we still need this 2nd outline?

Reply:
I did what @kumar303 suggested and used the default [focus mixin](https://github.com/mozilla/addons-frontend/blob/master/src/core/css/inc/mixins.scss#L92).